src/main/java/au/org/democracydevelopers/raireservice/request/GenerateAssertionsRequest.java

```diff
@@ -13,22 +13,13 @@ the raire assertion generation engine (https://github.com/DemocracyDevelopers/ra
 
 import au.org.democracydevelopers.raireservice.persistence.repository.ContestRepository;
 import java.util.List;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.data.annotation.ReadOnlyProperty;
 
```

Comment:
I'm still not sure what it means for a request to "describe a contest".

Reply:
Changed to "identifying the contest..."